docs/src/main/sphinx/connector/faker.md

```diff
@@ -260,37 +266,47 @@ CREATE TABLE generator.default.customer (LIKE production.public.customer EXCLUDI
 
 Insert random data into the original table, by selecting it from the
 `generator` catalog. Data generated by the Faker connector for columns of
-non-character types cover the whole range of that data type. Add constraints to
-adjust the data as desired. The following example ensures that date of birth
-and age in years are related and realistic values.
+non-character types cover the whole range of that data type. Set the `min` and
+`max` column properties, to adjust the generated data as desired. The following
+example ensures that date of birth and age in years are related and realistic
+values.
+
+Start with getting the complete definition of
+a table:
+
+```sql
+SHOW CREATE TABLE production.public.customers;
+```
+
+Modify the output of the previous query and add some column properties.
+
+```sql
+CREATE TABLE generator.default.customer (
+  id UUID NOT NULL,
+  name VARCHAR NOT NULL,
+  address VARCHAR NOT NULL,
+  born_at DATE WITH (min = '1900-01-01', max = '2025-01-01'),
+  age_years INTEGER WITH (min = '0', max = '150')
+);
+```
```

Comment:
Lets have an example of `values` in the docs as well

Reply:
Done! I've also noticed there's an outdated limitation at the end, I removed it in a separate commit.